Kickstarter-iOS/Views/Controllers/CommentsViewController.swift

```diff
@@ -28,3 +46,19 @@ internal final class CommentsViewController: UITableViewController {
 
   internal override func bindViewModel() {}
 }
+
+extension CommentsViewController {
```

Comment:
Cool! Just putting a note here to replace this with [cell card](https://kickstarter.atlassian.net/browse/NT-1892)